source/go-live/index.html.md.erb

```diff
@@ -1,14 +1,100 @@
+# Configure your service for production
 
-Before your service can go live, you must have [integrated with GOV.UK One Login's integration environment][integrate.integrate].
+Before you can configure your service for production, you must [integrate with GOV.UK One Login's integration environment][integrate.integrate].
+
+The process for configuring your service for production is: 
+
+1. Contact your Engagement Manager – if you do not have an Engagement Manager, [complete the form to register your interest](https://www.sign-in.service.gov.uk/register).
+1. Confirm with your Engagement Manager that you need to configure your service in production. Make sure you send the client ID of the client you’ve been testing in your integration configuration.
+1. The GOV.UK One Login team will send you a draft configuration in JSON format including the new client ID for your production service. 
+1. Fill in the JSON configuration by replacing the placeholder values with your service's configuration. There’s further [guidance on filling in your JSON configuration](/go-live/#fill-in-your-json-configuration). 
+1. Send your modified JSON configuration back to GOV.UK One Login by email. 
+1. The GOV.UK One Login team will check your production configuration and contact you if we need more information.
+1. Configure the new client ID into your own application code and deploy to your production environment. 
+1. Test your application works in production. This could be a limited test with a small number of users or a limited private beta.
+
+## Fill in your JSON configuration
+
+Use this table to help you fill in your JSON configuration.
+
+| Field | Notes |
+| :------ | :---- |
+| `BackChannelLogoutUri` | If you want to receive logout notifications from GOV.UK One Login, specify the production URI of the endpoint you want GOV.UK One Login to call.  <br><br>This must be a production-grade URI with domains without reference to `http://` and `localhost`. <br><br>There’s further guidance on [requesting logout notifications from GOV.UK One Login](/integrate-with-integration-environment/managing-your-users-sessions/#request-logout-notifications-from-gov-uk-one-login).  |
+| `ClientID` | GOV.UK One Login will fill in `ClientID` with your production client ID.  You do not need to do anything. |
+| `Claims` | If you’re doing identity verification, you’ll need to specify which claims your service requires.  You may choose one or more of the following:<br><ul> </li><li>[https://vocab.account.gov.uk/v1/passport](https://vocab.account.gov.uk/v1/passport) </li><li>[https://vocab.account.gov.uk/v1/drivingPermit](https://vocab.account.gov.uk/v1/drivingPermit) </li><li>[https://vocab.account.gov.uk/v1/coreIdentityJWT](https://vocab.account.gov.uk/v1/coreIdentityJWT) </li><li>[https://vocab.account.gov.uk/v1/address](https://vocab.account.gov.uk/v1/address) </li><li>[https://vocab.account.gov.uk/v1/returnCode](https://vocab.account.gov.uk/v1/returnCode)  |
```

Comment:
@PippaClarkGDS recommend removing the explicit links to the vocabulary to make it consistent with the rest of the tech docs and to avoid breaking the html-proofer with 404 page not found errors

Reply:
will do, thanks Paul